packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1674,20 +1697,21 @@ class StackViewport extends Viewport implements IStackViewport {
       );
 
       // store rotation cache since reset camera will reset it
-      const rotationCache = this.rotationCache;
+      const rotation = this.rotationCache;
 
       // Reset the camera to point to the new slice location, reset camera doesn't
       // modify the direction of projection and viewUp
       this.resetCameraNoEvent();
 
       // restore the rotation cache for the new slice
-      this.setRotation(rotationCache, rotationCache);
+      this.rotationCache = rotation;
```

Comment:
is `resetCameraNoEvent` changing the `this.rotationCache`? why we are again re assignging the rotation cache in the next line?

Reply:
Yes `resetCameraNoEvent` does change `this.rotationCache` as is mentioned in the comment on line 1699. That is why we are restoring it here.

Reply:
Hmm.. I should change the `const` to be `rotationCache`.